Content.Server/StationEvents/Events/CluwneBeastSpawn.cs

```diff
@@ -0,0 +1,35 @@
+using System.Linq;
+using Content.Server.StationEvents.Components;
+using Robust.Shared.Random;
+
+namespace Content.Server.StationEvents.Events;
+
+public sealed class CluwneBeastSpawn : StationEventSystem
+{
+    public static List<string> SpawnedPrototypeChoices = new List<string>()
```

Comment:
nice copy pasta

Reply:
yea, maybe i could have based a better example to base it off, i was worried the rev wouldnt take into account walls etc when spawning since it is a ghost and didnt want situations where cluwne beast spawns ontop of a wall. I noticed the adds a few mice thing in the ratking event and thought well having a few extra mice when an event triggers is no bad thing.

Reply:
have any suggestions for it?

Reply:
could always have the logic separate for cluwnebeast and rev/other threats that might go in walls

Reply:
I changed it and used a different events spawner for reference.